### DISCUSSION TITLE
VI keybindings?

### DISCUSSION THREAD

**Discussion by A7R7** [Taiko2k/Tauon#1011] [Q&A] [answered]
By default tauon music box uses left/right to cyle playlists, and up/down to select up/down track. 

Now I want to replace these keybindings with vi's, 'h' 'l' for cycling playlists, 'k' 'j' to select up/down track. But I meet 2 problems:
1. it seems there's is no option to tweak up/down selection in the input.txt file. 
2. Even if I set 'h' 'l' to cycle playlist, pressing them leads to global search.

Can I achieve this without introducing 3rd party apps or hacking into the keybinding system?

**Comment by Taiko2k** [comment, marked as answer]
Yeah the arrow keys are currently hard coded. Ill need to provision those to the input.txt file. Also currently due to the global search, binding letter keys requires a modifier key, but perhaps I could add a setting to disable that global search trigger. Ive opened issue #1012 to track this.

**Reply by A7R7** [reply]
Thank you for your quick answer and your awesome works!
